query-service-impl/src/test/resources/application.conf

```diff
@@ -292,5 +296,91 @@ service.config = {
         }
       }
     }
+    {
+      name = span-event-view-handler
```

Comment:
```suggestion
      name = span-event-view-trino-handler
```

Reply:
There is an already `pinot` handler type with name `span-event-view-handler`

Reply:
Yes, but `type` field will be different for both.

Reply:
Renamed handler, though I couldn't see any enforcement of handler name to be unique.